Jswope00/migrate learner docs (#828)
openedx/docs.openedx.org: source/learners/SFD_video_player.rst

.. _Video Player:

#######################################

> Watching Videos on the edX Video Player

Reviewer: ```suggestion
Watching Videos on the Open edX Video Player
```

#######################################

> All edX courses include videos, and edX has its own video player. Most of the

Reviewer: ```suggestion
Most courses include videos, and the Open edX platform has its own video player. Most of the
```

controls on the player will be familiar to you if you have watched online

> videos before. However, the edX video player has some extra features you may

Reviewer: ```suggestion
videos before. However, the Open edX video player has some extra features you may
```

not have seen.

The following image shows the video player in the middle of a video, followed
by an explanation of each option or control on the video player.

.. note:: The illustration that follows shows the video player for a video
   that has all of the possible features available. Only the controls that
   apply to a specific video will actually be present when you use the video
   player in a course.

.. image:: /_images/learners/Video_Intro.png
  :width: 800

>   :alt: The edX video player with numbered call-outs for every possible

Reviewer: ```suggestion
  :alt: The Open edX video player with numbered call-outs for every possible
```

    captions and the transcript with this control if translations are
    available.

    .. image:: /_images/learners/Video_language_menu.png
      :alt: A language menu with choices for English or Chinese.
      :width: 150

11. **Download video**: You can download the video to watch later with this
    option (if available for the video).

12. **Download transcript**: You can download the video's transcript as a
    SubRip (.srt) file or as a text (.txt) file. You can open files of either
    type in a text editor such as Notepad.

13. **Handouts**: You can download any handouts that the course team has
    associated with the video.

14. **License or Copyright**: If the course team reserves rights or specifies a
    Creative Commons license for the video that is different from the
    course-wide license, that information appears below the video player. For
    Creative Commons licenses, you can select the license to open a web site
    with more information about your rights.

    For more information, see :ref:`Course and Video Licenses`.

15. **Transcript**: You can scroll through the transcript and select any line
    to go to that point in the video.

> For a demonstration of the edX video player, see `Videos on edX`_ in the edX

Reviewer: ```suggestion
For a demonstration of the Open edX video player, see `Videos on Open edX`_ in the Open edX
```

Reviewer: Also make sure this links to the correct demo course
